lib/cli_common/cli_common/pulse.py

```diff
@@ -45,8 +45,22 @@
         connection_global=False
     )
 
-    queue = 'queue/{}/{}'.format(user, exchange)
+    exchange_name = exchange
+    if exchange.startswith(f'exchange/{user}/'):
+        exchange_name = exchange[len(f'exchange/{user}/'):]
+    elif exchange.startswith(f'exchange/'):
+        exchange_name = exchange[len(f'exchange/'):]
+
+    if not exchange.startswith('exchange/'):
+        exchange = f'exchange/{exchange}'
+
+    queue = f'queue/{user}/{exchange_name}'
```

Comment:
For pulse listener this will change from `queue/release-services-production/exchange/taskcluster-queue/v1/task-group-resolved` to `queue/release-services-production/taskcluster-queue/v1/task-group-resolved`, which will make us connect to a different queue than before. Could we keep using `exchange` instead?

Reply:
:+1: